pypesto/objective/aggregated.py

```diff
@@ -129,6 +131,18 @@ def get_config(self) -> dict:
             info[f"objective_{n_obj}"] = obj.get_config()
         return info
 
+    def create_history(
+        self, id: str, x_names: Sequence[str], options: HistoryOptions
+    ):
+        """See `history.generate.create_history` documentation."""
+        amici_objectives = [obj for obj in self._objectives if
+                            isinstance(obj, AmiciObjective)]
+        if len(amici_objectives) == 1:
```

Comment:
```suggestion
        if not amici_objectives
```
Otherwise you don't cover the cases where we have multiple amici objectives?

Reply:
Additionally (just hypothetical, what if we have mixtures of both amici and non amici objectives, will there be any clashes?